app/app/settings.py

```diff
@@ -696,3 +697,10 @@
             'name': 'Index View',
         }]
     SILKY_MAX_RECORDED_REQUESTS_CHECK_PERCENT = env.int('SILKY_MAX_RECORDED_REQUESTS_CHECK_PERCENT', default=10)
+
+# Gitcoin Bounty Funding Fee settings
+FEE_ADDRESS = env('FEE_ADDRESS', default='')
+FEE_ADDRESS_PRIVATE_KEY = env('FEE_ADDRESS_PRIVATE_KEY', default='')
```

Comment:
is it possible to test this on the mainnet, a la `FEE_ADDRESS_NETWORK`?

Reply:
Sure, though I don't think you need to specify a special network setting in `.env` for that.  You would just need to put an address and key with some mainnet ERC-20 balance in it, set the overall network in the .env for Mainnet, and then run the management command from the docker shell.